### PR TITLE
Show wait time and reverse scan in slow log

### PR DESCRIPTION
**Pull request by breeswish**
In addition, added special string `[slow_query]` for easier grep.

### CODE REVIEW

**Comment by BusyJay** [comment]
What can we get from the wait time?

**Comment by breeswish** [comment]
@BusyJay It can tell us how long the request has been blocked so that we can learn whether or not outdated mechanism is working as intended, as requested by @zhangjinpeng1987

**Comment by huachaohuang** [comment]
LGTM

File: src/coprocessor/endpoint.rs [hunk 1]
```diff
@@ -398,13 +399,17 @@ impl Drop for RequestTracker {
             };
 
             info!(
-                "[region {}] handle {:?} table id {:?} [{}] takes {:?} [keys: {}, hit: {}, \
-                 ranges: {} ({:?})]",
+                "[region {}] [slow_query] execute takes {:?}, wait takes {:?}, \
```
Comment (BusyJay) [inline src/coprocessor/endpoint.rs]: Prefer using `slow-query` here, so that it will be the same as tidb.

**Pull request merged**